test/suites/common-tanssi/services-payment/test_services_payment credit_buying.ts

```diff
@@ -0,0 +1,88 @@
+import "@tanssi/api-augment";
```

Comment:
Remove this file, it is a duplicate of `test/suites/common-tanssi/services-payment/test_services_pament_credit_buying_free_combined.ts`